### DISCUSSION TITLE
Prisma enviorment related error

### DISCUSSION THREAD

**Discussion by jas4f** [prisma/prisma#25888] [Q&A]
### Bug description

 I want to unit testing the shopify app code functions with the help of library "jest", but for this i changed test enviorment to "jest-environment-jsdom" . and see the below error :-
 **PrismaClient is unable to run in this browser environment, or has been bundled for the browser (running in `Node.js`).
so, please solved my problem**

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @jas4f 

The error you're encountering is related to using PrismaClient in a browser-like environment, which is not supported.
The problem stems from setting your Jest test environment to `jest-environment-jsdom`. This environment simulates a browser, which is incompatible with PrismaClient.

To solve this issue, you should change your Jest test environment to the default `node` environment. You can also look at this previously closed [issue](https://github.com/prisma/prisma/issues/25193#issuecomment-2338401087).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
